UnitsNet.Tests/CustomCode/ForceTests.cs

```diff
@@ -7,33 +7,36 @@ namespace UnitsNet.Tests
 {
     public class ForceTests : ForceTestsBase
     {
+        private const double StandardGravity = 9.80665;
+        private const double PoundsConversionRate = 4.4482216152605;
+
         protected override bool SupportsSIUnitSystem => true;
         protected override double DecanewtonsInOneNewton => 1E-1;
         protected override double DyneInOneNewton => 1E5;
 
-        protected override double KilogramsForceInOneNewton => 0.101972;
+        protected override double KilogramsForceInOneNewton => 1.0 / StandardGravity;
 
-        protected override double KilopoundsForceInOneNewton => 0.22481e-3;
+        protected override double KilopoundsForceInOneNewton => 1.0 / (PoundsConversionRate * 1_000.0);
         protected override double MeganewtonsInOneNewton => 1E-6;
         protected override double KilonewtonsInOneNewton => 1E-3;
 
-        protected override double KiloPondsInOneNewton => 0.101972;
+        protected override double KiloPondsInOneNewton => 1.0 / StandardGravity;
 
         protected override double NewtonsInOneNewton => 1;
 
         protected override double PoundalsInOneNewton => 7.23301;
 
-        protected override double PoundsForceInOneNewton => 0.22481;
+        protected override double PoundsForceInOneNewton => 1.0 / PoundsConversionRate;
 
-        protected override double TonnesForceInOneNewton => 1.019716212977928e-4;
+        protected override double TonnesForceInOneNewton => 1.0 / (StandardGravity * 1_000.0);
 
         protected override double MillinewtonsInOneNewton => 1.0e3;
 
         protected override double MicronewtonsInOneNewton => 1.0e6;
 
-        protected override double OunceForceInOneNewton => 3.596943089595368;
+        protected override double OunceForceInOneNewton => 1.0 / (PoundsConversionRate / 16.0);
 
-        protected override double ShortTonsForceInOneNewton => 1.12404471549816e-4;
+        protected override double ShortTonsForceInOneNewton => 1.0 / (PoundsConversionRate * 2_000.0);
 
```

Comment:
I understand your thinking to make the values more readable, but the tests are intentionally using constant values looked up in reference sources like a conversion table, wolfram alpha, online converter or similar. 

This way we sanity check any calculation described in the JSON files instead of doing the same calculation twice.

Reply:
As with the other suggestion, I was expecting this process to be the other way around, sorry.

Reply:
After discussing this further internally, I can't say we agree with this request. However, it is your call but would like you to consider #1309 prior to making a final decision.

Reply:
Until we conclude otherwise in #1309, let's merge this PR with improvements to the conversion constants. It is an improvement regardless of what changes to the test setup we end up with, and this PR is almost ready to go.

We can spend more time discussing the unit testing setup in general in #1309 .

The library guideline favors calculations in JSON conversion functions to preserve the highest precision possible, while favoring finite constants (less precise) in the test assertions to double check the conversion function based on a secondary source of documentation, such as wikipedia, wolfram alpha, etc. 

I think this is a sound approach, but I'm open to learning otherwise.
Let's have that discussion in #1309.